### DISCUSSION TITLE
Couldn't open a socket

### DISCUSSION THREAD

**Discussion by idinak** [milvus-io/milvus#3777] [Q&A and General discussion]
Hello,

I'm trying to startup milvus on RHEL 7.7 VM. It was difficult but I builded source code succesfully. When I start the milvus server gettin below exception.

![exception](https://user-images.githubusercontent.com/38636241/93341755-28d29300-f837-11ea-86ac-405f3ada5923.PNG)

ipv6.disable=1 default policy for ipv6.

How  to resolve this issue?

Thanks
Suleyman

**Comment by BossZou** [comment]
@idinak  This is because the thirdparty of http use the ipv6  family, which may cause issue if the machine disable ipv6. I have pull a request https://github.com/milvus-io/milvus/pull/3778 to solve this problem. After merged, it will be cherry pick to master barnch.
